SwiftyTesseract/SwiftyTesseract/SwiftyTesseract.swift

```diff
@@ -186,7 +190,64 @@ public class SwiftyTesseract {
     completionHandler(swiftString)
     
   }
-
+  
+  /// Takes an array UIImages and returns the PDF as a data object
+  ///
+  /// - Parameter images: Array of UIImages that should be analyzed
+  /// - Returns: PDF as Data object
+  /// - Throws: SwiftyTesseractError in case something went wrong
+  public func createPDF(from images: [UIImage]) throws -> Data {
+    let _ = semaphore.wait(timeout: .distantFuture)
+    
+    // create unique file path
+    let file = URL(fileURLWithPath: NSTemporaryDirectory()).appendingPathComponent(NSUUID().uuidString)
+    
+    guard let renderer = TessPDFRendererCreate(file.path, bundle.pathToTrainedData, 0) else {
```

Comment:
I think I would prefer pulling out the `TessPDFRendererCreate(_,_,_)` and `TessResultRendererBeginDocument(_,_)` logic into it's own function